### PR TITLE
add dynamodb example upon Naj's request

### PR DESCRIPTION
**Pull request by mvovchuk**

### CODE REVIEW

**Review by anelson — CHANGES_REQUESTED** [review]

File: dynamo-db-protect-and-restore-example/dynamoDBinstructions.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Backup DynamoDB Tables using Elastio
```
[the remark this answers is not in the document]
Comment (mvovchuk) [reply dynamo-db-protect-and-restore-example/dynamoDBinstructions.md]: done

File: dynamo-db-protect-and-restore-example/DynamoElastio.py [hunk 2]
```diff
@@ -0,0 +1,1334 @@
+#!/usr/bin/env python
```
[the remark this answers is not in the document]
Comment (mvovchuk) [reply dynamo-db-protect-and-restore-example/DynamoElastio.py:1]: @anelson https://github.com/bchew/dynamodump/blob/master/dynamodump/dynamodump.py this is the original, I am not sure how to include it - the original Licence is https://github.com/bchew/dynamodump/blob/master/LICENSE

[the remark this answers is not in the document]
Comment (mvovchuk) [reply dynamo-db-protect-and-restore-example/DynamoElastio.py:1]: done

File: dynamo-db-protect-and-restore-example/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+```
+elastio stream restore --rp rp-dg34vt9ouz4iphmo83hqxnno --to-file schema.json && python3 DynamoElastio.py -m restore -s Music --noConfirm
+```
+
+As a result you will find the table restored and ready for use.
+
+
+
+## LICENCE
```
Comment (anelson) [inline dynamo-db-protect-and-restore-example/README.md:72]:
```suggestion
## LICENSE
```

File: dynamo-db-protect-and-restore-example/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Elastio-specific modifications are Copyright 2022 Elastio Software, Inc, and are hereby released under the MIT License as was the original work upon which this version is derived (https://github.com/bchew/dynamodump).
```
Comment (anelson) [inline dynamo-db-protect-and-restore-example/README.md:74]: Let's make that hyperlink a bit prettier:

```suggestion
Elastio-specific modifications are Copyright 2022 Elastio Software, Inc, and are hereby released under the MIT License as was the [original work by bchew](https://github.com/bchew/dynamodump) from which this version is derived.
```

**Review by anelson — APPROVED** [review]
Please make requested changes prior to merge

**Pull request merged**